app/src/androidTest/java/org/mozilla/fenix/ui/SettingsBasicsTest.kt

```diff
@@ -86,45 +92,63 @@ class SettingsBasicsTest {
         }
     }
 
-    @Ignore("This is a stub test, ignore for now")
     @Test
     fun selectNewDefaultSearchEngine() {
-        // Open 3dot (main) menu
-        // Select settings
-        // Select "Search engine"
-        // Choose: DuckDuckGo
-        // Back arrow to Home
-        // Verify DuckDuckGo icon in Navigation bar
+        homeScreen {
+        }.openThreeDotMenu {
+        }.openSettings {
+        }.openSearchSubMenu {
+            enableDuckDuckGoAsSearchEngine()
+        }.goBack {
+        }.goBack {
+            verifyDefaultSearchEngineIsDDG()
+        }
     }
 
-    @Ignore("This is a stub test, ignore for now")
     @Test
     fun toggleSearchSuggestions() {
-        // Enter: "mozilla" in navigation bar
-        // Verify more than one suggesion provided
-        // Open 3dot (main) menu
-        // Select settings
-        // Select "Search engine"
-        // Toggle 'Show search suggestions' to 'off'
-        // Back arrow twice to home screen
-        // Enter: "mozilla" in navigation bar
-        // Verify no suggestions provided
+        homeScreen {
+        }.openNavigationToolbar {
+            verifySuggestionsAreMoreThan(1, "mozilla")
+        }.goBack {
+        }.openThreeDotMenu {
+        }.openSettings {
+        }.openSearchSubMenu {
+            disableShowSearchSuggestions()
+        }.goBack {
+        }.goBack {
+        }.openNavigationToolbar {
+            verifySuggestionsAreEqualTo(0, "mozilla")
+        }
     }
 
-    @Ignore("This is a stub test, ignore for now")
     @Test
     fun toggleShowVisitedSitesAndBookmarks() {
-        // Visit 3 static sites
-        // Bookmark 2 of them
-        // Open 3dot (main) menu
-        // Enter navigation bar and verify visited sites appear
-        // Verify bookmarks exist
-        // Open 3dot (main) menu
-        // Select settings
-        // Select "Search engine"
-        // Toggle off "Show visited sites and bookmarks"
-        // Back arrow twice to home screen
-        // Verify history and bookmarks are gone
+        val webPage = getGenericAsset(mockWebServer, 1)
+        val secondWebPage = getGenericAsset(mockWebServer, 2)
+        val thirdWebPage = getGenericAsset(mockWebServer, 3)
+
+        homeScreen {
+        }.openNavigationToolbar {
+        }.enterURLAndEnterToBrowser(webPage.url) {
```

Comment:
let's remember to include verification after each visit 
`verifyPageContent(page1.content)`